### PR TITLE
Attempt to further reduce dependecy duplication

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

**Comment by alex** [comment]
Looks like the old version of `tox` doesn't support this properly. @reaperhulk what's involved in upgrading `tox` on the `centos64`, `centos7`, `debian7`, and `freebsd10` builders?

**Comment by reaperhulk** [comment]
I'm already looking at it :)

**Comment by reaperhulk** [comment]
Jenkins, retest this please.

File: tox.ini [hunk 1]
```diff
@@ -5,11 +5,7 @@ envlist = py26,py27,pypy,py33,py34,py35,docs,pep8,py3pep8
 # If you add a new dep here you probably need to add it in setup.py as well
```
Comment (reaperhulk) [inline tox.ini]: This comment (and the one in setup.py referencing the tox.ini) are outdated now.

Comment (alex) [inline tox.ini]: done

**Pull request merged**